tests/integration_tests/test_tracegraph_elbo.py

```diff
@@ -349,21 +344,27 @@ def setUp(self):
         # gamma prior hyperparameter
         self.beta0 = Variable(torch.Tensor([1.0]))
         self.n_data = 2
-        self.data = Variable(torch.Tensor([3.0, 2.0]))  # two observations
-        self.alpha_n = self.alpha0 + \
-            Variable(torch.Tensor([self.n_data]))  # posterior alpha
-        self.beta_n = self.beta0 + torch.sum(self.data)  # posterior beta
+        self.data = Variable(torch.Tensor([[3.0], [2.0]]))  # two observations
+        self.alpha_n = self.alpha0 + self.n_data  # posterior alpha
+        self.beta_n = self.beta0 + self.data.sum(0)  # posterior beta
         self.log_alpha_n = torch.log(self.alpha_n)
         self.log_beta_n = torch.log(self.beta_n)
 
+    @pytest.mark.skipif(not dist.gamma.reparameterized, reason='not implemented')
+    def test_elbo_reparameterized(self):
+        self.do_elbo_test(True, 8000, 0.95, 0.0007)
```

Comment:
and here?

Reply:
ditto: I'd like tune parameters in a follow-up PR